process_environment.py

```diff
@@ -9,14 +9,14 @@
 
 DEBUG = False
 
-envIrradiance_cmd = "envIrradiance"
-envPrefilter_cmd = "envPrefilter"
-envIntegrateBRDF_cmd = "envBRDF"
-cubemap_packer_cmd = "cubemapPacker"
-panorama_packer_cmd = "panoramaPacker"
-envremap_cmd = "envremap"
-samplesGGX_cmd = "samplesGGX"
-envBackground_cmd = "envBackground"
+envIrradiance_cmd = "./envIrradiance"
```

Comment:
I will stick with a shell encapsulator :)
 "./" in path in VM might create some sublte bugs I'd prefer to avoid (bad
practice anyway)

On Fri, Jan 22, 2016 at 2:27 PM Cedric Pinson <notifications@github.com>
wrote:

> In process_environment.py
> <https://github.com/cedricpinson/envtools/pull/3#discussion_r50535910>:
>
> > @@ -9,14 +9,14 @@
> >
> >  DEBUG = False
> >
> > -envIrradiance_cmd = "envIrradiance"
> > -envPrefilter_cmd = "envPrefilter"
> > -envIntegrateBRDF_cmd = "envBRDF"
> > -cubemap_packer_cmd = "cubemapPacker"
> > -panorama_packer_cmd = "panoramaPacker"
> > -envremap_cmd = "envremap"
> > -samplesGGX_cmd = "samplesGGX"
> > -envBackground_cmd = "envBackground"
> > +envIrradiance_cmd = "./envIrradiance"
>
> export PATH to add . in your bashrc should be enough
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cedricpinson/envtools/pull/3/files#r50535910>.
>